### ISSUE TITLE
TypeError: can't assign to property "splitPattern" on 0: not an object

### ISSUE THREAD

**Issue by gatopeich** [briangann/grafana-datatable-panel#121]
Hitting this issue when trying this panel on a Simple JSON datasource that works fine with regular Grafana tables:

TypeError: can't assign to property "splitPattern" on 0: not an object

**Comment by ex3t3r** [comment]
Getting this error as well.

Data source: Basic select query from a Postgresql DB that works fine with Grafana tables. 
Grafan 7.1.5 with Datatable 1.0.2
Table options enabled and disabled do not change behaviour.
No column styles, sorting, alias, etc defined.
Occurs when selecting Scrolling or Paging for Table display.
Effectively any combination of the options can b used. 

When "Refresh Dashboard" is clicked or the table contents are updated based on an auto-refresh timer the error is displayed as follows: 
"TypeError: can't assign to property "splitPattern" on 0: not an object
    in t
    in t
    in div
    in div
    in div
    in div
    in div
    in t
    in t
    in t
    in div
    in Wa
    in div
    in on
    in div
    in div
    in t
    in div
    in t
    in div
    in t
    in div
    in t
    in div
    in div
    in t
    in x
    in div
    in t
    in x
    in r
    in Unknown
    in t
    in Unknown
    in h
    in Unknown
    in Unknown
    in t
    in t
    in l"

**Comment by devcode2** [comment]
This works:

```
SELECT F.TABLENAME AS 'Facility',
	COUNT(attribute_6) AS 'N',
	CONCAT(ROUND(COUNT(attribute_6) / (SELECT COUNT(attribute_6) FROM tokens_693553 WHERE usesleft = 0) * 100,2), '%') as '%'
	FROM tokens_693553 T
	LEFT JOIN TABLENAME F ON F.facility_num = T.attribute_6
	WHERE usesleft = 0
    group by 1
    order by 2 DESC;
```


This shows the error:

```
select * from TABLENAME
where Item NOT BETWEEN 72 AND 78
  AND Item not like 'D%'
order by cast(Item as unsigned) ASC;
```

**Comment by briangann** [comment]
fixed in 1.0.3!
